test/e2e/i18n-data-fetching-redirect/index.test.ts

```diff
@@ -7,6 +7,12 @@ import webdriver from 'next-webdriver'
 describe('i18n-data-fetching-redirect', () => {
   let next: NextInstance
 
+  // TODO: investigate tests failures on deploy
```

Comment:
Do you want to merge this?

Reply:
Yeah this will need to be investigated separately